livingdoc-fixture-converter/src/main/java/org/livingdoc/fixture/converter/number/ByteConverter.kt

```diff
@@ -0,0 +1,6 @@
+package org.livingdoc.fixture.converter.number
+
+
+open class ByteConverter : AbstractNumberConverter<Byte>() {
```

Comment:
Not happy with the idea of `ByteConverter` and `ShortConverter`. Length restrictions are technical concepts. What happens if the domain expert enters a number larger than 255 for ByteConverter?
`cut.convert("123456789") --> 21 ?!?`
Not sure if an exception is the solution here. Why do we go for length restrictions in the first place? `BigInteger` and `BigDecimal` all the way?